### ISSUE TITLE
failed to fetch on `local/root` project coordinate 

### ISSUE THREAD

**Issue by dharrigan** [liquidz/antq#33]
Hi,

I pull in several other custom libraries by specifying the `:local/root` directive in my `deps.edn`. 

For example, in my `deps.edn`:

```
:base {:extra-deps {local-base/local-base {:local/root "/home/david/.clojure/libs/local.base" :deps/manifest :deps}}}
:rebel {:extra-deps {local-nrepl/local-nrepl {:local/root "/home/david/.clojure/libs/local.nrepl" :deps/manifest :deps}
                     local-rebel/local-rebel {:local/root "/home/david/.clojure/libs/local.rebel" :deps/manifest :deps}}
        :main-opts ["-m" "local.rebel.main"]}
```

When `antq` is run against the `deps.edn`, the following is shown:
```

|    :file |                   :name | :version | :latest-version |
|----------+-------------------------+----------+-----------------|
| deps.edn |   local-base/local-base |          | Failed to fetch |
|          | local-nrepl/local-nrepl |          | Failed to fetch |
|          | local-rebel/local-rebel |          | Failed to fetch |
```

Perhaps `:local/root` should be ignored as part of the resolution process?

Thank you!

-=david=-

**Comment by liquidz** [comment]
@dharrigan Thanks for your reporting!

> Perhaps :local/root should be ignored as part of the resolution process?

You're right. I'll fix to ignore.

**Comment by liquidz** [comment]
@dharrigan I fixed in `dev` branch.
Could you try?

* checkout `dev` branch
* run `make uberjar` to generate target/antq-standalone.jar
* run `java -jar /path/to/your/antq/repo/target/antq-standalone.jar` in your project directory

**Comment by dharrigan** [comment]
Hi,

Looks good!

```
❯ cd .clojure                                                                                                                                        
❯ java -jar $HOME/tmp/antq/target/antq-standalone.jar
All dependencies are up-to-date.
```

On master branch:

```
❯ cd .clojure                                                                                                                                        
❯ java -jar $HOME/tmp/antq/target/antq-standalone.jar
|    :file |                   :name | :version | :latest-version |
|----------+-------------------------+----------+-----------------|
| deps.edn |   local-base/local-base |          | Failed to fetch |
|          | local-nrepl/local-nrepl |          | Failed to fetch |
|          | local-rebel/local-rebel |          | Failed to fetch |
```

All good! Thank you for the prompt turnaround!

-=david=-

**Comment by liquidz** [comment]
@dharrigan Thanks for your confirmation!

I just released ver 0.7.4 #34
